test/integration/buildpod_admission_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
For future, `corev1` is the prefferred alias for this api.

Reply:
I think VSCode may have done that...